runtime/tests/third_party/dummy.device.toml

```diff
@@ -0,0 +1,79 @@
+schema = 1
+
+[device]
+name = "lightning.qubit"
+version = "v0.31.0"
+precision = ['float32', 'float64']
+
+[operations]
+observables = [
+	"NamedObs",
+	"HermitianObs",
+	"TensorObs",
+	"HamiltonianObs",
+]
+
+[[operations.gates]]
+full = [
+	"Identity",
+	"PauliX",
+	"PauliY",
+	"PauliZ",
+	"Hadamard",
+	"S",
+	"T",
+	"PhaseShift",
+	"RX",
+	"RY",
+	"RZ",
+	"Rot",
+	"CNOT",
+	"CY",
+	"CZ",
+	"SWAP",
+	"IsingXX",
+	"IsingXY",
+	"IsingYY",
+	"IsingZZ",
+	"ControlledPhaseShift",
+	"CRX",
+	"CRY",
+	"CRZ",
+	"CRot",
+	"SingleExcitation",
+	"SingleExcitationMinus",
+	"SingleExcitationPlus",
+	"Toffoli",
+	"CSWAP",
+	"DoubleExcitation",
+	"DoubleExcitationMinus",
+	"DoubleExcitationPlus",
+	"MultiRZ",
```

Comment:
Shouldn't we remove all of these for the dummy_device? This will apply to other fields too
```suggestion
```

Reply:
This stems from the fact that we are not using "full" gates at all. The only field that is at the moment of interest are the "matrix" gates. We can delete them, they have no effect at the moment.

Reply:
Also, the dummy device does implement these gates! It doesn't have the same semantics, but they are implemented.

Reply:
Agreed, it makes sense to list these gates as long as the device lib doesn't raise an error when the corresponding function is called.